### PR TITLE
[FIX] Standardized tabular_metadata.yaml syntax to conform to: f04b47ff

### PR DESCRIPTION
**Pull request by TheChymera**
https://github.com/bids-standard/bids-specification/commit/f04b47ff makes a number of things much more interesting, but if we're doing this, probably best done everywhere.

@effigies

### CODE REVIEW

**Review by tsalo — APPROVED** [review]
Thanks for catching this. These changes look good to me.

**Review by effigies — APPROVED** [review]

File: src/schema/rules/tabular_metadata.yaml [hunk 1]
```diff
@@ -1,17 +1,15 @@
 ---
-# scans.tsv
-- suffixes:
+scans.tsv:
```
Comment (effigies) [inline src/schema/rules/tabular_metadata.yaml:2]:
```suggestion
scans:
```

File: src/schema/rules/tabular_metadata.yaml [hunk 2]
```diff
@@ -1,17 +1,15 @@
...
   - .json
   entities:
     subject: required
-    # session is required if session is present in the dataset.
-    session: optional
-# sessions.tsv
-# This file may only exist if session is present in the dataset.
-- suffixes:
+    session: optional # session is required if session is present in the dataset.
+sessions.tsv: # This file may only exist if session is present in the dataset.
```
Comment (effigies) [inline src/schema/rules/tabular_metadata.yaml:11]:
```suggestion
sessions: # This file may only exist if session is present in the dataset.
```

**Review by effigies — COMMENTED** [review]
Actually, I think we probably don't want `.`s in our keys.

**Comment by TheChymera** [comment]
@effigies fixed?

**Comment by TheChymera** [comment]
@effigies could merge this? I'd need it for updating the validator loading since right now this file needs the old loading model whereas all the others need the new one.

**Pull request merged**